src/WebSocketManager/WebSocketHandler.cs

```diff
@@ -31,12 +30,12 @@ await SendMessageAsync(socket, new Message()
             {
                 MessageType = MessageType.ConnectionEvent,
                 Data = WebSocketConnectionManager.GetId(socket)
-            });
+            }).ConfigureAwait(false);
         }
 
         public virtual async Task OnDisconnected(WebSocket socket)
         {
-            await WebSocketConnectionManager.RemoveSocket(WebSocketConnectionManager.GetId(socket));
+            await WebSocketConnectionManager.RemoveSocket(WebSocketConnectionManager.GetId(socket)).ConfigureAwait(false);
```

Comment:
From my somewhat limited understanding of `async/await`, I get that it is best practice to use `ConfigureAwait(configureOnCapturedContext: false)` with every `await` expression (reference from *C# in Depth 3rd edition*, page 506-507).

And I understand that it could be helpful in the .NET Client, where it would be used in an application with a UI.

But does it make sense in the .NET library to do so? In what scenario would a deadlock occur?